.travis.yml

```diff
@@ -16,6 +16,11 @@
 # under the License.
 
 language: java
+
+jdk:
+  - oraclejdk8
```

Comment:
I would prefer openjdk instead of oracle. In fact, you already have another PR for another JIRA that does exactly that: https://github.com/apache/parquet-mr/pull/596/files If you agree, you could resolve this PR and JIRA in favour of the other one.

Reply:
Great catch @zivanfi. I've changed it to openjdk.

Reply:
But what about the other JIRA? Why do we have two JIRA-s and two PR-s?

Reply:
Ah, okay, I see, one of the PR-s is for parquet-format, the other one is for parquet-mr. Do we need two JIRA-s though? I would suggest either closing one as a duplicate of the other or explicitly mentioning the repo name in the JIRA titles, in which case having two separate JIRA-s is fine.

Reply:
I see how this can be confusing.

Okay, my first thought was having two JIRA tickets would be cleaner. Looking at it now, we should have done this in one sweep. I've marked the newest one as duplicate and linked it to the former ticket for traceability. Hope it is more clear now :-)

Reply:
I think it was a good idea to use two separate JIRA-s, just their titles were confusing. Thinking more about my suggestion to merge them I see that it was a bad idea - after all, they will be released in different versions of different components. I will fix the title of the JIRA-s and then we can continue with the dual-JIRA approach.

Reply:
Good point. Making use of the components would be a good solution as well. Let me know where I can help.